.travis.yml

```diff
@@ -33,6 +33,8 @@ install:
       pip install -r requirements2.txt;
     else
       pip install -r requirements3.txt;
+      wget http://launchpadlibrarian.net/151932048/libc6_2.17-0ubuntu5.1_i386.deb;
```

Comment:
So tried to fix with that line but it didn't fix it so I just make it install an old version and try to figure out how to fix it in another issue